From 8d3cadcd28b13c44f8d17facb19390d645e6b0fa Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 7 May 2021 17:08:14 +0200
Subject: [PATCH] config: Distinguish Balena build

So we know it's a build meant to work
with BalenaOS.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 configs/iot-gate-imx8_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/iot-gate-imx8_defconfig b/configs/iot-gate-imx8_defconfig
index af4877876a..0cf56e0a75 100644
--- a/configs/iot-gate-imx8_defconfig
+++ b/configs/iot-gate-imx8_defconfig
@@ -13,7 +13,7 @@ CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_SPL=y
 CONFIG_CSF_SIZE=0x2000
 CONFIG_SPL_TEXT_BASE=0x7E1000
-CONFIG_LOCALVERSION="-iot-gate-imx8-2.3"
+CONFIG_LOCALVERSION="-iot-gate-imx8-2.3 Balena build"
 CONFIG_FIT=y
 CONFIG_FIT_EXTERNAL_OFFSET=0x3000
 CONFIG_SPL_LOAD_FIT=y
@@ -102,4 +102,4 @@ CONFIG_CMD_UNZIP=y
 CONFIG_CMD_IMPORTENV=y
 CONFIG_PARTITION_UUIDS=y
 CONFIG_CMD_PART=y
-CONFIG_CMD_FS_UUID=y
\ No newline at end of file
+CONFIG_CMD_FS_UUID=y
-- 
2.17.1

